src/main/webapp/consentGroupDocuments/ConsentGroupDocuments.js

```diff
@@ -0,0 +1,127 @@
+import { Component, Fragment } from 'react';
+import { Documents } from "../components/Documents";
+import { DocumentHandler } from "../util/ajax";
+import { ConsentGroupKeyDocuments } from "../util/KeyDocuments";
+import { ConfirmationDialog } from "../components/ConfirmationDialog";
+import { h } from 'react-hyperscript-helpers';
+import '../index.css';
+
+ class ConsentGroupDocuments extends Component {
+
+   constructor(props) {
+    super(props);
+    this.state = {
+      documentsCollection: [],
+      keyDocuments: [],
+      additional: [],
+      showDialog: false,
+      action: '',
+      uuid: ''
+    };
+  }
+
+   componentDidMount() {
+    this.getAttachedDocuments();
+  }
+
+   getAttachedDocuments = () => {
+    DocumentHandler.attachedDocuments(this.props.attachmentsUrl, this.props.projectKey).then(resp => {
+      this.setKeyDocuments(JSON.parse(resp.data.documents));
+    }).catch(error => {
+      console.error(error);
+    });
+  };
+
+   setKeyDocuments = (documentsCollection) => {
+    const keyDocuments = [];
+    const additionalDocuments = [];
```

Comment:
These are constants that you are modifying. Should they be `let`s instead?

Reply:
I think this is OK bcs these are arrays and the values are worked with `push()` - `pop()`. Unless we need to redefine the type of the variable then `let` should be the way, if not `const` is like a standar.